CHANGELOG.md

```diff
@@ -2,6 +2,40 @@
 
 Mapbox welcomes participation and contributions from everyone.
 
+## Mapbox Navigation SDK 1.6.2 - November 19, 2021
+This is a patch release on top of `v1.x` which does not include changes introduced in `v2.x` and later.
+
+### Changelog
+#### Bug fixes and improvements
+- Added `PedingIntent` Android 12 flags support and fixed other Android 12 incompatibilities by upgrading transitive dependencies. [#5142](https://github.com/mapbox/mapbox-navigation-android/pull/5142)
+
+### Mapbox dependencies
+This release depends, and has been tested with, the following Mapbox dependencies:
+- Mapbox Maps SDK `v9.7.1` ([release notes](https://github.com/mapbox/mapbox-gl-native-android/releases/tag/android-v9.7.1))
+- Mapbox Navigation Native `v32.0.0`
+- Mapbox Core Common `v9.2.0`
+- Mapbox Java `v5.9.0-alpha.1` ([release notes](https://github.com/mapbox/mapbox-java/releases/tag/v5.9.0-alpha.1))
+- Mapbox Android Core `v5.0.0`
+- Mapbox Android Telemetry `v8.1.0`
+
+## Mapbox Navigation SDK 2.0.2 - November 18, 2021
+This is a patch release on top of `v2.0.0` which does not include changes introduced in `v2.1.0-beta.1` and later. The `v2.1.0-rc.1` remains the latest and most up-to-date release.
```

Comment:
```suggestion
This is a patch release on top of `v2.0.1` which does not include changes introduced in `v2.1.0-beta.1` and later. The `v2.1.0-rc.2` remains the latest and most up-to-date release.
```
By the way, why is changelog for v2.1.0-rc.2 not here?

Reply:
Looks like the draft is here https://github.com/mapbox/mapbox-navigation-android/pull/5123. Happy to add it to this PR instead though.

Reply:
cc @kmadsen